File: lecture_2/prime_numbers.js
function getPrimeNum(n) {
    let result = [];
    for (let i = 2; result.length < n; i++) {
        let flag = false;
        for (let counter = 2; counter < i; counter++) {
            if (i % counter === 0) {
                flag = true;
                break;
            }
        }
        if (flag === false) {
            result.push(i)
        };
    }
    return result;
}

console.time();
console.log(getPrimeNum(process.argv[2]));
console.timeEnd();

File: lecture_2/prime_numbers.py
import time


def get_prime_numbers(n):
    result = []
    num = 2

    while len(result) < n:
        flag = False
        for item in range(2, num):
            if num % item == 0:
                flag = True
                break
        if flag == False:
            result.append(num)
        num += 1

    return result


if __name__ == "__main__":
    start_time = time.perf_counter()
    get_prime_numbers(1000)
    elapsed_time = time.perf_counter() - start_time
    print(elapsed_time)
